wellregistry/registry/migrations/0004_lookup_tables.py

```diff
@@ -0,0 +1,171 @@
+                'unique_together': {('country_cd', 'state_cd')},
+            },
+        ),
+        migrations.CreateModel(
+            name='CountyLookup',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('county_cd', models.CharField(max_length=3)),
+                ('county_nm', models.CharField(max_length=48)),
+                ('county_max_lat_va', models.CharField(max_length=11)),
+                ('county_min_lat_va', models.CharField(max_length=11)),
+                ('county_max_long_va', models.CharField(max_length=12)),
+                ('county_min_long_va', models.CharField(max_length=12)),
+                ('county_max_alt_va', models.CharField(max_length=8)),
+                ('county_min_alt_va', models.CharField(max_length=8)),
+                ('county_md', models.CharField(max_length=8)),
+                ('country_cd', models.ForeignKey(db_column='country_cd', on_delete=django.db.models.deletion.DO_NOTHING, to='registry.CountryLookup')),
+                ('state_cd', models.ForeignKey(db_column='state_cd', on_delete=django.db.models.deletion.DO_NOTHING, to='registry.StateLookup')),
+            ],
+            options={
+                'db_table': 'county',
+                'unique_together': {('country_cd', 'state_cd', 'county_cd')},
+            },
+        ),
+        migrations.AlterField(
+            model_name='registry',
+            name='agency_cd',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to='registry.AgencyLovLookup'),
+        ),
+        migrations.AlterField(
+            model_name='registry',
+            name='alt_datum_cd',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to='registry.AltDatumDimLookup'),
+        ),
+        migrations.AlterField(
+            model_name='registry',
+            name='alt_units',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to='registry.UnitsDimLookup'),
+        ),
+        migrations.AlterField(
+            model_name='registry',
+            name='county_cd',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to='registry.CountyLookup'),
+        ),
+        migrations.AlterField(
+            model_name='registry',
+            name='horz_datum',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to='registry.HorzDatumDimLookup'),
+        ),
+        migrations.AlterField(
+            model_name='registry',
+            name='nat_aquifer_cd',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to='registry.NatAqfrLookup'),
+        ),
+        migrations.AlterField(
+            model_name='registry',
+            name='state_cd',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to='registry.StateLookup'),
+        ),

```

Comment:
I thought we were going to use service calls for county, state, and country? Has that been abandoned or it is to be refactored later?

Reply:
This was discussed last week during backlog grooming and sprint planning which I think you missed. We want to be able to do things the "Django way". Plus we decided implementing all of the lookups the same way made more sense.

Reply:
Ok, sound fine to me. Just checking. Yes, I think I was out then. Thanks for the update.